packages/core/contracts/l2/polygon/README.md

```diff
@@ -0,0 +1,42 @@
+# Polygon <> Ethereum State Transfer
```

Comment:
10/10 on this. pretty awesome stuff!

Reply:
We should use this as a source of information for `docs.umaproject.org` when documenting the L2 compatibility with UMA.